modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/MysqlSchemaCodegen.java

```diff
@@ -29,10 +29,10 @@
 
 import static org.openapitools.codegen.utils.StringUtils.underscore;
 
+@SuppressWarnings("unchecked")
```

Comment:
Don't know what this _suppress_ does, but hope it's for good reasons.

Reply:
In Java, generic types are "erased" at runtime, so if you cast like a `Map<String, Object>` to `Map<String, CodegenModel>`, some IDEs like IntelliJ will warn that the cast is unchecked. You should also see these warnings at compilation time.  You'll see these unchecked suppressions throughout our code because much of our ADT is Object typed and we extract into the concrete types. It's safe throughout because we manage the types privately, although it could fail if someone were to set a different type in an overridden method (but that would break everything anyway).

I'm hoping to improve our ADT and template binding models for 5.0 to be typed better.